### PR TITLE
WT-10073 timestamp_abort : Move checkpoint/flush_tier calls to test_util

### PR DESCRIPTION
**Pull request by raviprakashgiri29**

### CODE REVIEW

File: test/csuite/timestamp_abort/main.c [hunk 1]
```diff
@@ -336,27 +350,54 @@ thread_ckpt_run(void *arg)
...
+      ckpt_flush_config, sizeof(ckpt_flush_config), "flush_tier=(enabled,force),%s", ckpt_config));
+
+    set_flush_tier_delay(&td->extra_rnd);
+
     /*
      * Keep a separate file with the records we wrote for checking.
      */
     (void)unlink(ckpt_file);
+    memset(ckpt_flush_config, 0, sizeof(ckpt_flush_config));
+    memset(ckpt_config, 0, sizeof(ckpt_config));
```
Comment (ddanderson) [inline test/csuite/timestamp_abort/main.c:372]: I don't understand why `ckpt_flush_config` and `ckpt_config` are being cleared apparently after they are set above this.

**Review by ddanderson — COMMENTED** [review]
There's something I don't understand - the rest looks good.

File: test/csuite/timestamp_abort/main.c (hunk 1 above)
Comment (raviprakashgiri29) [reply test/csuite/timestamp_abort/main.c]: Sorry, it was a wrong commit. Fixed in 4c4e4764589a6f942ffdb2f1ab16cf870ee3459c

**Review by ddanderson — APPROVED** [review]
LGTM!

**Pull request merged**